addons/mail/static/src/core/common/composer.xml

```diff
@@ -235,4 +235,12 @@
             <t t-esc="option.label"/>
         </em>
     </t>
+    <t t-name="mail.Composer.suggestionEmoji">
+        <strong class="px-2 py-1 align-self-center flex-shrink-1 text-truncate">
+            <t t-esc="option.emoji.codepoints"/>
+        </strong>
+        <em class="text-600 text-truncate align-self-center" style="flex-basis: 20%;">
+            <t t-esc="option.emoji.shortcodes.join(', ')"/>
+        </em>
+    </t>
```

Comment:
Emoji codepoints is visually too small. fixed by removing `py-1` and adding `fs-3`